.github/workflows/upload-to-bucket.yml

```diff
@@ -0,0 +1,35 @@
+name: Upload to S3
+
+on: [pull_request]
+
+jobs:
+  upload:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: chrnorm/deployment-action@releases/v1
+        name: Create GitHub deployment
+        id: status
+        with:
+          token: ${{ secrets.GITHUB_TOKEN}}
+          description: 'Upload to S3'
+          environment: preview
+
+      - uses: shallwefootball/s3-upload-action@master
+        name: Upload S3
+        id: S3
+        with:
+          aws_key_id: ${{ secrets.AWS_KEY_ID }}
+          aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY}}
+          aws_bucket: ${{ secrets.AWS_BUCKET }}
+          source_dir: 'spark-jobs'
+          destination_dir: 's3://datateam-spark/jobs/'
```

Comment:
đã bucket rồi mà còn đưa nguyên cả bucket và path lên là sao

Reply:
AWS_BUCKET ở đây là name bucket thoi mà anh, còn cái destination_dir path kia thì là destination directory for upload á